src/components/routes/Host/ListingForm/ListingHelp.tsx

```diff
@@ -23,7 +23,10 @@ const HomeType = () => (
 const Title = () => (
   <div>
     <h3>Listing Name</h3>
-    <p>&quot;Beautiful Home with Wifi and Views&quot;</p>
+    <h3>Choose a name that perfectly describes your listing. Be sure to use proper casing.</h3>
+    <p>&quot;Beautiful Home with View of Manhattan Skyline&quot;</p>
+    <p>&quot;Convenient Room in the Heart of San Francisco&quot;</p>
+    <p>&quot;Downtown LA Apartment near Pershing Square&quot;</p>
```

Comment:
The fontography here gets a little funny with the two `h3`s

What about `<h3>` for Listing Name, `<p>` for "Choose a name...", then maybe a `<ul>` for the examples? That seems consistent with the other help text

Reply:
good idea. thanks!